config/src/main/resources/app-config.xml

```diff
@@ -19,9 +19,11 @@
         <baseUrl mapToProperty="dialogarena.cms" alias="appres.cms"/>
         <baseUrl mapToProperty="soknadsapi" alias="soknadsapi"/>
         <applicationProperties alias="soknadsosialhjelp.properties"/>
+        <credential alias="suspender" />
     </resources>
 
     <loadBalancer isAlive="/soknadsosialhjelp/internal/isAlive"/>
+    <!--<suspend url="/soknadforeldrepenger/management/suspend" credential="suspender" />-->
 
```

Comment:
Hva ville denne linjen gjort hvis den var kommentert inn? Kanskje legge inn en kommentar som forklarer hva den eventuelt skulle ha gjort?

Reply:
Den skal kommenteres inn etter første deploy. Men er et sirkulært problem at for å kunne deploye nedetidsfri, må appen støtte nedetidsfri. Så legger til støtten, men aktiverer det ikke ved første deploy, men så på neste. Om det ga mening? Haha

Reply:
Ok. Godkjenner.